features/submitted_at_inbox.feature

```diff
@@ -1,12 +1,16 @@
-@pipeline
-Feature: Sequencing pipelines should display submitted at date.
```

Comment:
Is there any reason for commenting this out rather than removing it?

Reply:
removing that